spec/Section 2 -- Language.md

```diff
@@ -986,7 +986,7 @@ field vs not altering a field, respectively. Neither form may be used for an
 input expecting a Non-Null type.
 
 Note: The same two methods of representing the lack of a value are possible via
-variables by either providing the a variable value as {null} and not providing
+variables by either providing the variable value as {null} and not providing
```

Comment:
The usage of "either" with "and" is probably a typo (https://english.stackexchange.com/a/70068).  
I would suggest changing it to "...by either providing the variable value as {null} **or** not providing...".